src/mmw/package.json

```diff
@@ -33,6 +33,7 @@
     "leaflet": "0.7.3",
     "leaflet-draw": "git://github.com/michaelguild13/Leaflet.draw#e2fe3a4",
     "leaflet.locatecontrol": "0.43.0",
+    "leaflet-plugins": "git://github.com/azavea/leaflet-plugins",
```

Comment:
Since this fork was made to make a small change to get browserify to work, can you submit a PR to the upstream and create a local issue so we can track it?  Would be best to eventually be able to remove this fork.